examples/speaker_recognition/README.txt

```diff
@@ -5,6 +5,8 @@ for voxceleb trial files you can download
 voxceleb1_test_v2.txt from http://www.openslr.org/resources/49/voxceleb1_test_v2.txt and
 veri_test2.txt from http://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/veri_test2.txt 
 
+Generate manifest files for voxceleb test files (see $NeMo/scipts to see how we can generate scp to manifest using simple steps) 
```

Comment:
"to see how we can generate scp to manifest using simple steps" <-- this grammar is weird. Generate scp?